modules/Bio/EnsEMBL/IO/Format.pm

```diff
@@ -309,7 +351,7 @@ sub validate_as_string {
 
 sub validate_as_integer {
   my ($self, $value) = @_;
-  return $value =~ /^-?\d+$/ ? undef : "Value $value is not an integer";;
+  return $value =~ /^-?\d+$/ ? 1 : 0;
```

Comment:
Note that this assumes we never get positive values with explicit sign, numbers with thousand separators and the like. Perhaps this is okay for our input - but still, I would really prefer using Regexp::Common here instead of reinventing the wheel and possibly getting bitten in the arse by edge cases.